pkgs/development/tools/misc/sloccount/default.nix

```diff
@@ -40,6 +41,14 @@ stdenv.mkDerivation rec {
     mkdir -p "$out/share/doc"
   '';
 
+  postInstall = ''
+    for w in "$out/bin"/*; do
+      if [ "$(file --brief --mime-encoding $w)" != "binary" ]; then
```

Comment:
We also have a `isELF()` function that could be used instead here.

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/auto-patchelf.sh#L209

Reply:
Thanks! Now I also saw that there's also an `isScript` so I'll use that I think.